frame/fast-unstake/src/tests.rs

```diff
@@ -35,16 +35,28 @@ fn test_setup_works() {
 #[test]
 fn register_works() {
 	ExtBuilder::default().build_and_execute(|| {
+		ErasToCheckPerBlock::<T>::put(1);
 		// Controller account registers for fast unstake.
 		assert_ok!(FastUnstake::register_fast_unstake(RuntimeOrigin::signed(2)));
 		// Ensure stash is in the queue.
 		assert_ne!(Queue::<T>::get(1), None);
 	});
 }
 
+#[test]
+fn register_unsigned_fails() {
```

Comment:
This test is not doing what the name says -- expecting unsigned origin? or rename the test.